.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,18 +1,18 @@
 ---
+
 name: Bug report
 about: Create a report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
----
-
-<!-- Please fill out as much of the template as you can. -->
+---<!-- Please fill out as much of the template as you can. -->
```

Comment:
It looks like the prettier-ing combined the first line and the end of the metadata (also in the other two issue template files)